### ISSUE TITLE
Start java language worker in placeholder mode

### ISSUE THREAD

**Issue by pragnagopa** [Azure/azure-functions-host#3776]
Tracking issue to warm up java language worker when using placeholders

**Comment by ColbyTresness** [comment]
To clarify this is the Java only work for now, correct?

**Comment by pragnagopa** [comment]
PR #3824 is scoped to Java for now. The goal is to expand to other languages as well.

**Comment by ColbyTresness** [comment]
What I'm really asking is: should I open a new issue specifically for JS or use this to track as well?

**Comment by pragnagopa** [comment]
synced up with @mhoeger offline. Once the PR is merged in, I will open tracking items for nodejs worker

**Comment by ColbyTresness** [comment]
reminder to open JavaScript tracking items :)
